mars/tensor/random/core.py

```diff
@@ -313,8 +313,9 @@ def _on_serialize_random_state(rs):
 
 
 def _on_deserialize_random_state(tup):
-    rs = np.random.RandomState()
-    rs.set_state(tup)
+    rs = np.random.RandomState(0)
+    if tup is not None:
```

Comment:
When the `tup` is None?

Reply:
when serializing `random_state`, if passed random state is  None, the `_on_serialize_random_state` just return None. In such situation,  the `tup` is `None`. I'm not sure why `random_state` is None, but I think it's a bug in previous and without this fix, `GroupBySample` shuffle on ray will fail. Mars tests won't fail just because it runs in test session which no serialziation happend. The exception is soemthing like follows and we can submmit another PR to fix it:
```
E         File "mars/serialization/core.pyx", line 924, in mars.serialization.core.deserialize
E           raise exc_value
E         File "mars/serialization/core.pyx", line 888, in mars.serialization.core.deserialize
E           deserialized = _deserial_single(
E         File "mars/serialization/core.pyx", line 802, in mars.serialization.core._deserial_single
E           res = serializer.deserial(serialized[_COMMON_HEADER_LEN:], context, subs)
E         File "mars/serialization/core.pyx", line 103, in mars.serialization.core.Serializer.deserial
E           cpdef deserial(self, tuple serialized, dict context, list subs):
E         File "/Users/chaokunyang/Desktop/chaokun/python/mars/mars/core/operand/base.py", line 338, in deserial
E           operand: Operand = super().deserial(serialized, context, subs)
E         File "/Users/chaokunyang/Desktop/chaokun/python/mars/mars/serialization/serializables/core.py", line 238, in deserial
E           self._set_field_value(obj, field, value)
E         File "/Users/chaokunyang/Desktop/chaokun/python/mars/mars/serialization/serializables/core.py", line 220, in _set_field_value
E           field.set(obj, field.on_deserialize(value))
E         File "/Users/chaokunyang/Desktop/chaokun/python/mars/mars/tensor/random/core.py", line 317, in _on_deserialize_random_state
E           rs.set_state(tup)
E         File "mtrand.pyx", line 357, in numpy.random.mtrand.RandomState.set_state
E       TypeError: state must be a dict or a tuple.
```

Reply:
Fixed it in #3149, I'll merge master after it's merged